package.json

```diff
@@ -8,7 +8,8 @@
     "main": "dist/oneui.min.js",
     "scripts": {
         "storybook": "start-storybook -s ./stories/static -p 9001 -c .storybook",
-        "storybook:prod": "build-storybook -s ./stories/static -c .storybook",
+        "storybook:build": "build-storybook -s ./stories/static -c .storybook",
+        "storybook:deploy": "storybook-to-ghpages --script=storybook:build --out=.out",
```

Comment:
![error](https://user-images.githubusercontent.com/7724351/52267668-f31ab800-2939-11e9-9d67-e807877f6302.png)

I can't deploy it. Am I missing something? Did you get this too?

Reply:
Seems to be caused by branch protection, now that I've turned that off for `gh-pages` it seems to work again

Reply:
Now it worked!